uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -0,0 +1,122 @@
+          </div>
+          <div class="col-md-6">
+            <button type="button" class="button__delete btn-danger">Delete User Account</button>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-8 contact_action_menu__body">
+        <div class="row">
+          <div class="col-md-4">
+            <h3>Leave:</h3>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-plus"></i></span> Record New
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-search"></i></span> View Entitlements
+              </button>
+            </p>
+            <hr>
+            <p><strong>Leave Approver(s):</strong> </p>
+            <p><a href="#" class="text-primary">John Snow</a></p>
+            <p><a href="#" class="text-primary">Allan Wite</a></p>
+            <button type="button" class="btn btn-secondary-outline">Manage Leave Approver</button>
+          </div>
+          <div class="col-md-4">
+            <h3>Workflows:</h3>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-user-plus"></i></span> Joining
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-user-times"></i></span> Exiting
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">Other...</button>
+            </p>
+            <hr>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-check-square-o"></i></span> New Task
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-id-card-o"></i></span> New Document
+              </button>
+            </p>
+            <hr>
+            <p><strong>Line Manager(s):</strong> </p>
+            <p><a href="#" class="text-primary">John Snow</a></p>
+            <p><a href="#" class="text-primary">Allan Wite</a></p>
+            <button type="button" class="btn btn-secondary-outline">Manage Line Manager</button>
+          </div>
+          <div class="col-md-4">
+            <h3>Communicate </h3>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-envelope-o"></i></span> Send Email
+              </button>
+            </p>
+            <p>
+              <button type="button" class="btn btn-primary-outline">
+                <span class="btn-icon"><i class="fa fa-users"></i></span> Record Meeting
```

Comment:
What is purpose of `btn-icon` class? There is no css written for it. And if it is really required, please follow BEM.

Reply:
will remove! not needed!